src/Altinn.Platform/Altinn.Platform.Storage/Storage/Models/Instance.cs

```diff
@@ -105,7 +105,7 @@ public class Instance
         /// Gets or sets the form ids associated with the instance
         /// </summary>
         [JsonProperty(PropertyName = "data")]
-        public Dictionary<string, Dictionary<string, Data>> Data { get; set; }
+        public Dictionary<string, Data> Data { get; set; }
 
```

Comment:
For this change, a new issue must be created to updated the service implementation for instance

Reply:
Have create issue #1581